src/CP15.cpp

```diff
@@ -306,7 +340,7 @@ void ARMv5::UpdateRegionTimings(u32 addrstart, u32 addrend)
             MemTimings[i][0] = bustimings[2] << NDS.ARM9ClockShift;
         }
 
-        if (pu & 0x10)
+        if (pu & CP15_MAP_DCACHEABLE)
         {
             MemTimings[i][1] = kDataCacheTiming;
```

Comment:
these values will only ever be used when JIT is enabled, right?

Reply:
Right now, yes. They are used in ARMv5::CodeRead32, when the JIT is not enabled.